harness/determined/experimental/core_v2/_core_v2.py

```diff
@@ -233,6 +260,15 @@ def init(
     """
     Core V2 initializer in the singleton style.
     """
+
+    if defaults is not None or unmanaged is not None:
+        warnings.warn(
+            "Determined.experimental.core_v2.init is deprecating `defaults` and `unmanaged`."
```

Comment:
nit: wording, maybe something like
```
'defaults' and 'unmanaged' parameters have been deprecated and will be removed in a future version. Please use `config` instead.
```

to be more in line with our current deprecation messages